spec/spec.md

```diff
@@ -457,11 +457,57 @@ The following properties are for use at the top-level of the resource — all
 other properties that are not defined below MUST be ignored:
 
 - `name` - The resource ****MAY**** contain this property, and if present its
-value ****SHOULD**** be a human-friendly name that describes what the
-Presentation Definition pertains to.
+  value ****SHOULD**** be a human-friendly name that describes what the
+  Presentation Definition pertains to.
 - `purpose` - The resource ****MAY**** contain this property, and if present its
-value ****MUST**** be a string that describes the purpose for which the
-Presentation Definition's inputs are being requested.
+  value ****MUST**** be a string that describes the purpose for which the
+  Presentation Definition's inputs are being requested.
+- The resource ****MAY**** include a `format` property, and its value 
+  ****MUST**** be an object with one or more properties matching the registered 
+  [Credential Format Designations](#credential-format-designations) to inform the 
+  Holder of the credential format configurations the Verifier can process. The 
+  value for each property included ****MUST**** be an object composed as follows:
+    - The object ****MAY**** include a format-specific property (i.e. `alg`, `type`) 
+      that expresses which algorithms the Verifier supports for the format, and if 
+      present, its value ****MUST**** be an array of one or more of the 
+      format-specific algorithmic identifier references, as noted in the 
+      [Credential Format Designations](#credential-format-designations) section.
+
+      ```json
+      {
+        "presentation_definition": {
+          "format": {
+            "jwt": {
+              "alg": ["EdDSA", "ES256K", "ES384", "PS256"]
+            },
+            "jwt-vc": {
```

Comment:
do you think these "alg" values should contain different algs to demonstrate they don't all have to be the same?

Reply:
Could help, but I don't see why in practice you would support different ones for different JWTs

Reply:
legacy JWT OIDC servers that use outdated RSA crypto... just because one dependency uses an old alg doesn't mean to you should allow that everywhere...